_docs/_latest/configure/scanner/rules.md

```diff
@@ -867,3 +867,52 @@ rules:
     the JMESPath expression in `key` extracts an integer, you probably
     want integers in this list.  Similarly, if the expression extracts
     a list of values, you need to provide lists.
+
+## Retention rules
+
+### Rule definition
+
+```yaml
+rules:
+  - name: retention policies should be in a specific range
```

Comment:
the rule should be more descriptive

maybe 'all buckets should have a retention policy for 100 to 200 days.